hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -402,6 +402,7 @@ message OMResponse {
   optional SetSnapshotPropertyResponse       SetSnapshotPropertyResponse   = 128;
   optional ListStatusLightResponse           listStatusLightResponse       = 129;
   optional SnapshotInfoResponse              SnapshotInfoResponse          = 130;
+  optional OMLockDetailsProto                omLockDetailsProto            = 131;
```

Comment:
omLockDetailsProto > omLockDetails

Reply:
For this one, since we also have a class named "OMLockDetails", the removal of the "Proto" will cause some confusion.

Reply:
Updated.